doc/install_macos.txt

```diff
@@ -0,0 +1,197 @@
+.. include:: css.txt
+
+.. _install_macos:
+
+
+Mac OS Installation Instructions
+################################
+
+There are various ways to install Theano dependencies on a Mac. Here
+we describe the process in detail with Canopy, Anaconda, Homebrew or
+MacPorts but if you did it differently and it worked, please let us
+know the details on the `theano-users`_ mailing-list, so that we can
+add alternate instructions here.
+
+.. _theano-users: http://groups.google.com/group/theano-users?pli=1
+
+In academia: Enthought Canopy
```

Comment:
I think it is important to keep conda in the installation instruction. That way, for people that have different linux OS, it is easy to do it. It is also a very nice way to install everything as a user that don't have root access.

I would kept enthought if the instruction still work, but if it get complicated to maintain, we can discuss to remove it. I would put conda before enthough.